tests/integration/ipsec_test.py

```diff
@@ -893,6 +893,41 @@ def test_ipsec_ipv4_libreswan_p2p_cert_auth_add_and_remove(
     )
 
 
+@pytest.mark.xfail(
+    reason="NetworkManager-libreswan might be too old",
```

Comment:
Instead of giving a vague reason, can you mention the earliest version that NetworkManager-libreswan supports leftsubnet? 
Based on the NEWS file, https://gitlab.gnome.org/GNOME/NetworkManager-libreswan/-/blob/1d3182af8ffa04981c703095508c4e279737de11/NEWS, the version is NetworkManager-libreswan-1.2.20.

Reply:
Other than this, the patch looks good to me.

Reply:
Sure thing!